### PR TITLE
Multiple fixes

### PR DESCRIPTION
**Pull request by sevein**
This PR attempts to fix:
- https://github.com/artefactual/archivematica-docs/issues/136
- https://github.com/artefactual/archivematica-docs/issues/139
- https://github.com/artefactual/archivematica/issues/1065

### CODE REVIEW

File: admin-manual/installation-setup/installation/install-ubuntu.rst [hunk 1]
```diff
@@ -100,8 +100,15 @@ Ubuntu 16.04 (Xenial) installation instructions
...
-      sudo python get-pip.py
+      curl -Ls https://bootstrap.pypa.io/get-pip | sudo python - "pip==9.0.3"
+
+   Notice that we are using ``pip 9.0.3``, this is important if you are using
+   Ubuntu 14.04 (see `issue #1065`_). Ubuntu 16.04 works with more recent
+   versions of ``pip`` so you could run the following instead:
+
+   .. code:: bash
+
+      curl -Ls https://bootstrap.pypa.io/get-pip | sudo python -
```
Comment (mamedin) [inline admin-manual/installation-setup/installation/install-ubuntu.rst]: This change is for Ubuntu Trusty section below, this is the Ubuntu Xenial section.

**Review by mamedin — CHANGES_REQUESTED** [review]

File: admin-manual/installation-setup/installation/install-ubuntu.rst (hunk 1 above)
Comment (sevein) [reply admin-manual/installation-setup/installation/install-ubuntu.rst]: Fixed!!! :D

**Comment by sevein** [comment]
Please review again!

**Review by mamedin — APPROVED** [review]
LGTM

**Pull request merged**